src/bootstrap.jl

```diff
@@ -193,6 +193,22 @@ function allpars(bsamp::MixedModelFitCollection{T}) where {T}
     )
 end
 
+function StatsBase.confint(bsamp::MixedModelBootstrap{T}; level::Real=0.95) where {T}
+    cutoff = sqrt(quantile(Chisq(1), level))
+    # Creating the table is somewhat wasteful because columns are created then immediately skipped.
+    tbl = Table(bsamp.tbl)
```

Comment:
we do have a `shortestcovint(bsamp::MixedModelFitCollection{T}, level=0.95)` method that we could use and then filter

Reply:
It's quite possible that the logic in that section is more convoluted than necessary - a not uncommon problem for me.  Let me look at it afresh to see if I can figure out what I was thinking.

Reply:
Ah, I see what the issue is.  It has to do with the names of the parameters in the Table representation of the bootstrap results relative to the `allpars` representation.  In the allpars representation there are three columns whose combined values determine the parameter, which can be something like $\sigma$, `batch`, `(Intercept)` whereas in the Table (property name `tbl`) that would be just $\sigma1$.  Eventually I think the parameter names should just be symbols like $\beta1$, $\sigma1$, ... .  A separate extractor can return the mapping from symbol to parameter type, grouping factor, column designator.  Trying to embed all that information in each row of the .allpars table just created a mess.

Regarding using `shortestcovint` applied to the bootstrap sample to get the intervals, that goes through the .allpars construction, which is probably more long-winded than the .tbl construction.  It might be best to just add an optional argument to the .tbl construction so that it skips the $\theta$ and `obj` columns.